packages/reactor-api/src/processors/analytics-processor.ts

```diff
@@ -13,11 +15,9 @@ export abstract class AnalyticsProcessor<
 
   constructor(args: ProcessorSetupArgs, options?: ProcessorOptions) {
     super(args, options);
-    this.analyticsStore = args.dataSources.analyticsStore;
-  }
-
-  onSetup(args: ProcessorSetupArgs) {
-    super.onSetup(args);
-    this.analyticsStore = args.dataSources.analyticsStore;
+    this.analyticsStore = new KnexAnalyticsStore({
+      executor: new KnexQueryExecutor(),
+      knex: args.operationalStore,
```

Comment:
Not sure what this does. Why is the operational store passed to the analytics store?

Is this something that needs to be done by each processor or can just be done once in the reactor-api?